Sources/Orbit/Support/Text/LocalizedStringKey.swift

```diff
@@ -0,0 +1,84 @@
+import SwiftUI
+
+public extension Bundle {
+    
+    func localized(locale: Locale) -> Bundle {
+        let bundlePath = path(forResource: locale.identifier, ofType: "lproj") 
+            ?? path(
+                forResource: locale.identifier.split(separator: "_").map(String.init).first ?? locale.identifier, 
+                ofType: "lproj"
+            )
+        return (bundlePath.map(Bundle.init(path:)) ?? self) ?? self
+    }
+}
+
+public extension LocalizedStringKey {
+
+    private typealias FArgs = (CVarArg, Formatter?)
```

Comment:
`FormatterArguments`? I really can't tell what this short name is supposed to be... 😅

Reply:
Renamed to `FormattedVarArgs`